### PR TITLE
Jdoba backend routing

### PR DESCRIPTION
**Pull request by josephdoba**
Its officially hitting the db! 

Create:
create

Edit:
edit

placed the "Address" field to ``tables.sql``, so you will need to retable/seed the db

I'm aware that at the moment, owner id and party limit id' are showing up as null, which would cause new created events and edit events not to render. That fix will come in the next push.

### CODE REVIEW

File: client/joinus_client/src/components/Events/EventCard.js [hunk 1]
```diff
@@ -37,14 +47,60 @@ export default function EventCard(props) {
     reload,
   } = props;
 
-  const { userCreateEventSubmit } = userEvents();
+  console.log("Props from EventCard.js:")
+  console.log(props)
+
+  // to duck: We need an address props - Where would they come from? the db.. yea but.. what fil... the db??.. sigh, what I mean is, what is the parent of where these props are coming in? why do I need to know? They're coming in from useAppData as a hook -joba
```
Comment (carmtsang) [inline client/joinus_client/src/components/Events/EventCard.js:53]: what do you mean?

File: client/joinus_client/src/components/Events/EventCard.js [hunk 2]
```diff
@@ -193,8 +250,206 @@ export default function EventCard(props) {
...
+        sx={FormBox}
+        onSubmit={(event) => {
+          event.preventDefault();
+          const sendDataObj = {
+            eventName,
+            eventAddress,
+            eventImage,
+            eventDescription,
+            eventSizeLimit: 2,
+            eventOwnerId: user.id, // grab owner_id from cookies .. just user user.id?
```
Comment (carmtsang) [inline client/joinus_client/src/components/Events/EventCard.js:276]: the user's id should be set, if it is not, grab it from the localStorage

File: client/joinus_client/src/components/Events/EventCard.js [hunk 3]
```diff
@@ -193,8 +250,206 @@ export default function EventCard(props) {
...
+            eventCategory,
+            lat: 51.0233064354121, // will eventually need to generate these values from address
```
Comment (carmtsang) [inline client/joinus_client/src/components/Events/EventCard.js:278]: these values should automatically generate from the address...

**Review by carmtsang — COMMENTED** [review]
see comments

**Review by carmtsang — APPROVED** [review]
will need a run through of this after

File: client/joinus_client/src/components/Events/EventCard.js (hunk 1 above)
Comment (josephdoba) [reply client/joinus_client/src/components/Events/EventCard.js:53]: This is my own ramblings -ignore this haha

File: client/joinus_client/src/components/Events/EventCard.js (hunk 3 above)
Comment (josephdoba) [reply client/joinus_client/src/components/Events/EventCard.js:278]: Refer to: `// will eventually need to generate these values from address`

File: client/joinus_client/src/components/Events/EventCard.js [hunk 4]
```diff
@@ -37,14 +47,60 @@ export default function EventCard(props) {
...
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    "& > :not(style)": { m: 1, width: "100%" },
+  };
 
   const [open, setOpen] = useState(false);
   const [error, setError] = useState(false);
 
+  // Form info State declarations:
```
Comment (josephdoba) [inline client/joinus_client/src/components/Events/EventCard.js:80]: These are needed for the edit form to function

**Pull request merged**